### PR TITLE
Update UI images in Discussion Forum

### LINKED ISSUE #664: Remove previous web UI images
Out of date images on the [discussion](https://docs.synapse.org/articles/discussion.html) page.  :(

### PR DESCRIPTION
**Pull request by kelshmo**
Fixes #664

### CODE REVIEW

File: _articles/discussion.md [hunk 1]
```diff
@@ -11,56 +11,42 @@ category: collaboration-and-communication
...
+The Discussion tab can be found in the Project navigation tabs on the Project main page. Anyone with Project access may view discussion threads contained in the Project, create new threads, or reply to existing threads.
 
-## Creating a New Message Thread
+<img style="width: 100%;" src="../assets/images/Discussion_Main.png">
 
-The `Discussion` tab can be found in the project navigation tabs on your project's main page. Anyone with `Project` access may view discussion threads contained in the project, create new threads, or reply to existing threads.
+To create a new thread, click the **New Thread** button at the top of the Discussion tab.
 
-<img style="width: 65%;" src="../assets/images/Discussion Main.png">
+<img style="width: 100%" src="../assets/images/NewThread_Dialogue.png">
```
Comment (Aryllen) [inline _articles/discussion.md:24]: This image is broken when I try to run this locally. May want to double check that it's working.

File: _articles/discussion.md [hunk 2]
```diff
@@ -11,56 +11,42 @@ category: collaboration-and-communication
...
-
-In case of accidental deletion, moderators may also undelete threads to restore them. To view deleted threads, click on "Show Deleted Threads" from the main forum page to see a list of threads that are currently in the trash can. Once you've found the thread you'd like to restore, click into that thread and then use the Tools Menu and select "Restore Thread".
-
-## Pinning Threads
-
-Moderators may also "pin" important threads using the **pin** icon located next to the original post within the thread page. This causes the thread to be highlighted on the top of the thread list on the `Discussion` main page, and to remain at the top of the list even once newer threads are posted. Moderators may also "unpin" threads.
-
-<img style="width: 65%;" src="../assets/images/ModeratorButtons.png">
+Moderators may emphasize important threads by selecting the **pin** icon to bring the post to the top of the forum. The thread will remain at the top of the list even once newer threads are posted. Moderators may also unpin threads.
```
Comment (Aryllen) [inline _articles/discussion.md:49]:
```suggestion
Moderators may emphasize important threads by selecting the **pin** icon to bring the post to the top of the forum. The thread will remain at the top of the list even after newer threads are posted. Moderators may also unpin threads.
```

**Review by Aryllen — CHANGES_REQUESTED** [review]
One image issue and one grammar suggestion.

File: _articles/discussion.md (hunk 1 above)
Comment (kelshmo) [reply _articles/discussion.md:24]: oops that is because it wasn't committed! That should be fixed now.

**Review by kelshmo — COMMENTED** [review]
@Aryllen I added the missing image!

**Review by Aryllen — APPROVED** [review]
Thanks, @kelshmo!

**Pull request merged**